openid-4-verifiable-presentations-1_0.md

```diff
@@ -687,6 +758,11 @@ This document also defines the following additional error codes and error descri
 
 - The Presentation Definition URL can be reached, but the specified `presentation_definition` cannot be found at the URL.
 
+`invalid_request_uri_mode`:
+
+- The value of the `request_uri_mode` request parameter is neither `get` nor `post`.
```

Comment:
I don't know what is a value for `get` and `post` to be insensitive but if they are, this sentence should probably reflect this.

Reply:
added (case-insensitive)